infrastructure/k8s/cache-apt.tf

```diff
@@ -73,6 +73,7 @@ resource "kubernetes_deployment" "cache-apt" {
                     name = "cache-apt"
                     gce_persistent_disk {
                         pd_name = google_compute_disk.cache-apt.name
+                        read_only = false
```

Comment:
That's an excellent question. It would sometimes fail. The commit message has a big question mark because I *think* it solves it, but I'm not sure. I can remove this commit from the PR if you think it's dubious (I do think it's dubious ...)

Reply:
That's odd. I mean, unless `read_only` defaults to false I would expect it to always fail (not just sometimes). I think the problem is somewhere else.

I don't mind the commit, though.